### PR TITLE
fix README.md examples

### PR DESCRIPTION
**Pull request by iHmD**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ $app->get('/foo', function ($req, $res, $args) {
   //Here you expect 'username' and 'age' parameters
   if($this->validation->hasErrors()){
     //There are errors, read them
-    $errors = $this->getErrors();
+    $errors = $this->apiValidation->getErrors();
```
Comment (DavidePastore) [inline README.md]: Here it's `$this->validation` and not `$this->apiValidation`

**Comment by DavidePastore** [comment]
Hi @iHmD , thanks for this PR! I've added a comment on it. Can you also fix the same error in [line 129](https://github.com/DavidePastore/Slim-Validation/pull/2/commits/99029a82ba863efdb7cf9fd01012c0abecaf88b4?diff=unified#diff-04c6e90faac2675aa89e2176d2eec7d8R129)?

**Comment by iHmD** [comment]
that's  weird!
it gave me an error saying  "Call to a member function ... on a non-object ...", and after I added the service it worked! Maybe it depended on my situation though :)

**Comment by iHmD** [comment]
@DavidePastore I pushed a new commit with new fixes !

File: README.md [hunk 2]
```diff
@@ -126,7 +126,7 @@ $app->post('/bar', function ($req, $res, $args) {
   //Here you expect 'username' and 'age' parameters
   if($this->validation->hasErrors()){
     //There are errors, read them
-    $errors = $this->getErrors();
+    $errors = $this->->validation->getErrors();
```
Comment (DavidePastore) [inline README.md]: Be careful here: there is an additional `->` :ice_cream:

**Comment by iHmD** [comment]
@DavidePastore hope it is **OK** this time!

**Comment by DavidePastore** [comment]
Yes @iHmD , it's OK! Thanks for this PR and congratulations for your first one! :+1:

**Comment by iHmD** [comment]
@DavidePastore  thank you very much :smile:

**Pull request merged**